code-challenges.js

```diff
@@ -26,20 +26,43 @@ const fruit2 = "banana"
 
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
-// Expected output: 9
+const worldSeriesRuns = [2 ,5 , 2 , 2, 4 ,6 ,3 ,5 ,3]
```

Comment:
You don't want to just hard code a combined array. To combine these 2 arrays, you can use the built in method `.concat`.